src/middlewared/middlewared/plugins/system_advanced/config.py

```diff
@@ -259,6 +261,10 @@ async def do_update(self, data):
             if original_data['motd'] != config_data['motd']:
                 await self.middleware.call('etc.generate', 'motd')
 
+            if original_data['login_banner'] != config_data['login_banner']:
+                await self.middleware.call('etc.generate', 'ssh')
```

Comment:
Also you probably don't need to do this anymore because on `service.action` we generate the service's related etc group